README.md

```diff
@@ -93,9 +93,6 @@ hellosign.signatureRequest.send({/*options*/})
     .catch(function(err){
         //do something with error
     })
-    .finally(function(){
```

Comment:
Thinking about this, my change is a breaking change, since the `when.js` promise probably has slightly different behaviour / functionallity than what is in the ES spec.

Also this could break for (very) old versions of node

We can look here for what versions of node support `promise.finally`
https://node.green/#ES2018-features-Promise-prototype-finally

Or here for native promises
https://node.green/#ES2015-built-ins-Promise

Reply:
Great callout/consideration. As of now, the only areas that have been updated within the codebase are the only ones that I see as potentially affected by when.js. The additional lines you've added to still handle the promise should properly handle them accordingly. As you've mentioned, our tests are currently all passing and I ran some additional tests as well to confirm callback functionality was working correctly. I have completed my initial review. @nathanbuchar